src/object_/lifetime.d

```diff
@@ -0,0 +1,749 @@
+/**
+ * Contains utilities for managing lifetimes.
+ *
+ * This module is not intended to be imported or used directly.  It is
+ * publicly imported by object.d, and should be accessed through there.
```

Comment:
What does "should be accessed through there" mean? Would it be more accurate to say that the declarations in this module are automatically available in all D modules, like those in `object`?

Reply:
> This module is not intended to be imported or used directly.  It...

"It" refers to "This module". 

> It is publicly imported by object.d, and should be accessed through there.

"there" refers to "object.d"

This is natural English for me (using pronouns to refer to previously mentioned things).  If you'd like me to change it, please send me copy-pastable text.

Reply:
What I'm trying to say is that it might not be obvious to users that they don't have to import anything in order to use the symbols from that module. E.g. someone googles for "dlang destroy object", finds this page on dlang.org, sees that it's in a module, and tries importing it. Heck, at that point the user might not even know that even `object` doesn't need to be imported.

"accessed through there" also can be interpreted that something needs to be done by the user to achieve this, specifically something involving what "there" refers to (the `object` module).

Reply:
I gave it another go to address your concerns.  I don't know if it's good enough.  Please send me something better it's not.

Reply:
It's better?